tests/e2e/utils/constants.go

```diff
@@ -5,6 +5,7 @@ package utils
 
 const (
 	baseDir               = ".avalanche-cli"
+	baseAPMDir            = ".apm"
```

Comment:
can't we use constants in `pkg/constants/constants` for this?

Reply:
fixed